features/enable_fips_cloud.feature

```diff
@@ -546,14 +301,17 @@ Feature: FIPS enablement in cloud based machines
         Examples: ubuntu release
            | release | fips-name    | fips-service |fips-apt-source                                |
            | bionic  | FIPS         | fips         |https://esm.ubuntu.com/fips/ubuntu bionic/main |
+           | bionic  | FIPS Updates | fips-updates |https://esm.ubuntu.com/fips/ubuntu bionic/main |
+           | focal   | FIPS         | fips         |https://esm.ubuntu.com/fips/ubuntu focal/main  |
+           | focal   | FIPS Updates | fips-updates |https://esm.ubuntu.com/fips/ubuntu focal/main  |
 
     @slow
     @series.bionic
+    @seris.focal
```

Comment:
Running through the integration tests has proved worth it! - I found a tiny typo :) @lucasmoura 
```suggestion
    @series.focal
```

Reply:
Great catch. I have fixed it now